wiki/Rules/en.md

```diff
@@ -1,77 +1,79 @@
+5.  **Where the rules do not prevail, common sense shall.** The administration has explicit discretion to apply their judgment on this as they see fit.
+
+In-Game Chat Rules
+------------------
+
+1.  **Be respectful and mindful of other users in the channel, and avoid flooding or spamming.** One person taking up half the screen at once is annoying, so avoid doing it wherever possible. Other people are trying to chat too!
+2.  **Let the moderators handle issues in the chat.** You can point things out to them with the `!report` command and they'll handle the rest.
+3.  **If someone gets timed out, silenced or restricted, leave them be and don't start talking about them.** If someone is restricted, their case involves only them and the game administration - absolutely nobody else!
+4.  **Encourage fair play, and don't give attention to cheaters or other toxic users.** These people unfortunately do exist, and giving them attention can encourage them to continue, or provoke others to act as they do. Report them via the chat command `!report` or the buttons on their profile page and move on.
+5.  **Keep the chat clean and appropriate for an all-ages community, avoiding NSFW topics where possible.** NSFW stuff really isn't appropriate for an all-ages community. Do what you like in private messages (within reason) but keep the public channels squeaky clean for everyone's sake.
+6.  **Be mindful of what you share with others, especially if the content is potentially disturbing or very graphic in nature.** If you have to think twice about whether something is appropriate to link, it probably isn't okay. Avoid things with fast, strobing lights, gore, images of abuse or other graphic material.
+7.  **Keep links to relevant and legal websites.** Bootleg streaming websites (officially licensed ones like Animelab and Crunchyroll are OK) and anything related to "warez" or pirated software of any kind aren't okay, and we don't want them in our chat. Advertising Twitch links for other video games is okay, but ONLY in `#videogames`.
+8. **Don't advertise third-party chat servers of any kind, including Discord, Teamspeak or Skype servers/groups.** The osu! chat is about osu! foremost and not about sharing links to your personal servers. Do it via PM if you absolutely have to.
+9.  **In language-specific channels, you must speak the dominant language (English for #english, Thai for #thai, etc) if you are participating in discussion.** Where the language for a channel is not explicitly stated (or inferred from the channel name, as in Thai for `#thai` and Dutch for `#dutch`), it is always English. The `#lobby` and `#help` channels can be used with any and every language, unlike other channels. *Members of the moderation staff](/wiki/People/Global_Moderation_Team) do not need to follow this rule when doing their work.*
```

Comment:
uh, what happened? markdown derp
replace last sentence with this:
`*Members of the [moderation staff](/wiki/moderation_staff) do not need to follow this rule when doing their work.*`